bravado/client.py

```diff
@@ -271,15 +282,16 @@ def response_callback(incoming_response, operation):
     raise_on_unexpected(incoming_response)
 
     try:
-        swagger_return_value = unmarshal_response(incoming_response, operation)
+        incoming_response.swagger_result = unmarshal_response(
```

Comment:
Yeah, I'll document that as part of #178. It is not exactly relevant to the public interface in this PR.